Assets/Plugins/CountlySDK/Services/OptionalParametersCountlyService.cs

```diff
@@ -2,13 +2,16 @@ namespace Plugins.CountlySDK.Services
 {
     public class OptionalParametersCountlyService
     {
+        private readonly RecordLocationService _recordLocation;
         public string CountryCode { get; private set; }
```

Comment:
these location fields shouldn't exist anymore

Reply:
these are part of prv module. if i remove them. you can't set IP. contrcode and city all of them after them following functions.
setIpAddres 
setCountryCode
setCity
note: last time when we deprecate these functions you raised this question too.

Reply:
remove the "private set" an link the "get" with the new module